src/main/webapp/WEB-INF/tags/instructor/home/feedbackSessionActions.tag

```diff
@@ -0,0 +1,138 @@
+    data-toggle="tooltip"
+    data-placement="top">
+  Edit
+</a>
+<a class="btn btn-default btn-xs btn-tm-actions session-delete-for-test margin-bottom-7px<c:if test="${not actions.allowedToDelete}"> disabled</c:if>"
+    href="${actions.allowedToDelete ? actions.deleteLink : 'javascript:;'}"
+    title="<%= Const.Tooltips.FEEDBACK_SESSION_MOVE_TO_RECOVERY %>"
+    data-toggle="tooltip"
+    data-placement="top"
+    data-courseid="${actions.courseId}"
+    data-fsname="${actions.fsName}">
+  Delete
+</a>
+<div title="<%= Const.Tooltips.FEEDBACK_SESSION_COPY %>"
+    data-toggle="tooltip"
+    data-placement="top"
+    style="display: inline-block; padding-right: 5px;">
+  <a class="btn btn-default btn-xs btn-tm-actions session-copy-for-test margin-bottom-7px"
+      href="javascript:;"
+      data-actionlink="${actions.editCopyLink}"
+      data-courseid="${actions.courseId}"
+      data-fsname="${actions.fsName}"
+      data-toggle="modal"
+      data-target="#fsCopyModal"
+      id="button_fscopy-${actions.courseId}-${actions.fsName}">
+    Copy
+  </a>
+</div>
+<div title="<%= Const.Tooltips.FEEDBACK_SESSION_SUBMIT %>"
+    data-toggle="tooltip"
+    data-placement="top"
+    style="display: inline-block; padding-right: 5px;">
+  <a class="btn btn-default btn-xs btn-tm-actions session-submit-for-test margin-bottom-7px<c:if test="${not actions.allowedToSubmit}"> disabled</c:if>"
+      href="${actions.allowedToSubmit ? actions.submitLink : 'javascript:;'}">
+    Submit
+  </a>
+</div>
+<div title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTS %>"
+    data-toggle="tooltip"
+    data-placement="top"
+    style="display: inline-block; padding-right: 5px;">
+  <div class="btn-group margin-bottom-7px">
+    <a class="btn btn-default btn-xs btn-tm-actions session-view-for-test"
+        href="${actions.resultsLink}">
+      Results
+    </a>
+    <button type="button"
+        class="btn btn-default btn-xs btn-tm-actions dropdown-toggle session-results-options"
+        data-toggle="dropdown"
+        aria-expanded="false"
+        <c:if test="${not actions.publishButton.actionAllowed}">disabled</c:if>>
+      <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu" role="menu">
+      <li>
+        <a class="session-view-for-test" href="${actions.resultsLink}">View Results</a>
+      </li>
+      <li>
+        <tif:feedbackSessionPublishButton publishButton="${actions.publishButton}" showTooltip="false"/>
+      </li>
+      <c:if test="${actions.allowedToResendPublishedEmail}">
+      <li>
+        <a href="javascript:;"
+            data-actionlink="${actions.sessionResendPublishedEmailPageLink}"
+            class="session-resend-published-email-for-test"
+            data-courseid="${actions.courseId}"
+            data-fsname="${actions.fsName}"
+            data-toggle="modal"
+            data-target="#resendPublishedEmailModal">
+          Resend link to view results
+        </a>
+      </li>
```

Comment:
Should the `<li>` part be indented one level further than the `<c>` tag?

Reply:
Oh didn't notice that. I just copy paste `feedbacks/feedbackSessionActions.tag`. Will change both.